### DISCUSSION TITLE
Mount multiple Google drive accounts

### DISCUSSION THREAD

**Discussion by Drezzler** [Studio-42/elFinder#3335] [Q&A]
Right now, as i see it , only one google drive account can be mounted.

Can some, please tell me how to mount multiple google drive accounts ?

Any help is much appreciated.

**Comment by nao-pon** [comment]
@Drezzler You can additionally authenticate other accounts by clicking the circle icon.

![210610-152326](https://user-images.githubusercontent.com/1412630/121475338-3c92da80-ca00-11eb-8f03-220e10563369.png)
